settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from dotenv import load_dotenv
+from pydantic.v1 import BaseSettings
+
+
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

Comment:
The `DATABASE_URL` is set to a default SQLite database. Ensure that this is the intended database for your application, or adjust it to match your production database settings.